_data/other.yml

```
@@ -567,6 +567,13 @@ websites:
       exceptions:
           text: "Two factor authentication unavailable for 'End User' access, which is read-only."
 
+    - name: Weclapp 
+      url: https://weclapp.com/
+      img: weclapp.png
+      tfa: Yes
+      software: Yes
+      doc: https://www.weclapp.com/de/blog/2-faktor-authentifizierung/
```

The blog post describes U2F implementation which we categorize as hardware 2FA.
```suggestion
      hardware: Yes
      doc: https://www.weclapp.com/de/blog/2-faktor-authentifizierung/
```